### ISSUE TITLE
change `$_` to `_`

### ISSUE THREAD

**Issue by michaelficarra** [larryng/coffeescript-repl#1]
`$_` seems like an odd choice. In the official REPL (and in node's), the result of the last evaluation is stored as `_`.

**Comment by larryng** [comment]
I chose $_ as to not override the popular jQuery's $ nor Underscore.js's _.
On May 12, 2012 8:26 PM, "Michael Ficarra" <
reply@reply.github.com>
wrote:

**Comment by michaelficarra** [comment]
`_` is really the proper choice here. I think it's what everyone would expect. We haven't gotten a single complaint for this behaviour in the official REPL:

```
$ coffee
coffee> _ = require 'underscore'
{ [Function]
  _: [Circular],
  VERSION: '1.3.3',
  /* ... [snip] ... */
  template: [Function],
  chain: [Function] }
coffee> 0
0
coffee> _
0
coffee> 
```

**Comment by larryng** [comment]
I agree that `_` is what is expected, but Underscore.js came first and I'm not quite ready to break compatibility with the library to save a character.  In the future, I'll likely allow the user to change the variable in a setting, once there are Settings.

In the meantime, you are welcome to change it on your own local copy or fork.  It is literally a one character change [here](https://github.com/larryng/coffeescript-repl/blob/master/js/main.js#L47).

EDIT: With the recent rewrite, make that [here](https://github.com/larryng/coffeescript-repl/blob/master/coffee/main.coffee#L10).
